### DISCUSSION TITLE
Welcome to the community!

### DISCUSSION THREAD

**Discussion by BSick7** [nullstone-io/nullstone#65] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you are building with Nullstone.
Feel free to join our [community slack](https://nullstone-community.slack.com/join/shared_invite/zt-1vxmvd55t-2sbVk3e7cHqX1UbS17qjXg#/shared-invite/email).

**Comment by henryl** [comment]
How do we join the community slack? It requires a nullstone domain.

**Reply by ssickles** [reply]
Hi, sorry about that. It appears the settings for the Slack workspace got messed up. Here is the correct link.
https://nullstone-community.slack.com/join/shared_invite/zt-1vxmvd55t-2sbVk3e7cHqX1UbS17qjXg#/shared-invite/email

I also updated the link above.
